sdk/runanywhere-python/runanywhere/_generated_errors.py

```
@@ -0,0 +1,165 @@
+    UNSPECIFIED = 0
+    NETWORK = 1
+    VALIDATION = 2
+    MODEL = 3
+    COMPONENT = 4
+    IO = 5
+    AUTH = 6
+    INTERNAL = 7
+    CONFIGURATION = 8
+
+
+__all__ = ["ErrorCode", "ErrorCategory"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)